CONTRIBUTING.md

```diff
@@ -15,36 +15,49 @@
 
 2. Fork the dynatrace-operator repository and get the source code:
 
-```sh
-git clone https://github.com/<your_username>/dynatrace-operator
-cd dynatrace-operator
-```
+    ```sh
+    git clone https://github.com/<your_username>/dynatrace-operator
+    cd dynatrace-operator
+    ```
 
 3. Install development prerequisites:
 
-```sh
-make prerequisites
-```
+   ```sh
+   make prerequisites
+   ```
 
 4. Create a new branch to work on:
 
-```sh
-git checkout -b feature/your-branch
-```
+   ```sh
+   git checkout -b feature/your-branch
+   ```
 
 5. Once the changes are finished, make sure there are no warnings on the code. For debugging you can [run the unit tests](#unit-tests) and [end-to-end tests](#e2e-tests).
 
-> **NOTE:**
-> Unit tests are always executed via pre-commit hook (installed on previous steps). Meaning, you can only commit code that passes all unit tests.
+    > **NOTE:**
+    > Unit tests are always executed via pre-commit hook (installed on previous steps). Meaning, you can only commit code that passes all unit tests.
 
-```sh
-make go/test
-make test/e2e/<scope_of_the_changes>
-```
+    ```sh
+    make go/test
+    make test/e2e/<scope_of_the_changes>
+    ```
+
+6. To test your changes on a cluster use
+
+    1. kubectl to connect to a cluster
+    2. Use make commands to build and deploy your operator as follows:
+
+    ```sh
+    make build && make deploy
+    ```
+
+    >**NOTE:**
+    > When building on ARM machines (such as Apple M#) podman./docker uses the local architecture if not specified otherwise.
```

Comment:
```suggestion
    > When building on ARM machines (such as Apple M1) podman/docker uses the local architecture if not specified otherwise.
```

it would be great to add an example:
```
OPERATOR_DEV_MAC=linux/amd64 make images/build/push
```

Reply:
```
OPERATOR_DEV_BUILD_PLATFORM
```
wdyt to rename it?

cc @0sewa0  need third opinion on that

Reply:
okey based on vote, we are good to rename it. So please rename to `OPERATOR_DEV_BUILD_PLATFORM`